modules/oe_whitelabel_starter_event/config/install/core.entity_view_display.node.oe_sc_event.teaser.yml

```diff
@@ -54,3 +54,4 @@ hidden:
   oe_content_short_title: true
   oe_documents: true
   oe_featured_media: true
+  search_api_excerpt: true
```

Comment:
And another thing I forgot to mention.
We are changing the appearance of date range teaser, but we are not changing the formatter or showing the time.
Do we actually have a spec for this?

Reply:
The Event teaser uses short dates in the PDF, but instead of "to" the separator is "-". For this reason I didn't change anything.

Reply:
I do see the "Event teaser" part in the pdf now. It does use hyphen with spaces.
I see that we don't need to add the spaces with the separator here, the core formatter already adds the spaces.
So it seems ok.

Can we add a line in ContentEventRenderTest::testEventRenderingTeaser() to test the date output? I think we currently don't have it.